packages/components/src/components.d.ts

```diff
@@ -45,13 +44,11 @@ export { AlertType, AlertVariant, KoliBriAlertEventCallbacks } from "./types/ale
 export { KoliBriColor } from "./components/badge/component";
 export { Stringified } from "./types/common";
 export { KoliBriHorizontalIcon, KoliBriIconProp } from "./types/icon";
-export { AlternativButtonLinkRole, AriaCurrent, ButtonProps, KoliBriButtonCallbacks, KoliBriButtonType, KoliBriButtonVariant, LinkOnCallbacks, LinkTarget, LinkUseCase } from "./types/button-link";
-export { NavLinkProps } from "./components/link/component";
+export { AlternativButtonLinkRole, AriaCurrent, ButtonProps, KoliBriButtonCallbacks, KoliBriButtonType, KoliBriButtonVariant, LinkOnCallbacks, LinkProps, LinkTarget, LinkUseCase } from "./types/button-link";
```

Comment:
So wie ich das lese, braucht ein export .. from .. kein import vorher. Sofern die Imports in der Datei selber gebraucht werden wäre export .. ausreichend.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/export